### ISSUE TITLE
Aria roles out of scope in accordion menu.

### ISSUE THREAD

**Issue by MichellanneLi** [zurb/foundation-sites#9223]
There is a problem with how aria roles are applied to foundation's accordion menu, and it impacts that accessibility of the menu.

I discovered this problem when running the a11y command line accessibility auditor. To replicate the steps I took, please do the following:
1. Install the a11y command line accessibility auditing tool. Instructions can be found here: https://github.com/addyosmani/a11y
2. In the terminal, run the command $a11y https://michellanneli.github.io/okeefelaw/index.html (This is the prototype of a website I have been working on, in which the accordion menu is implemented for mobile.)
3. Note the following error message: 
   
   ✖ Elements with ARIA roles must be in the correct scope
   # js-main-nav > .vertical.menu.main-nav.accordion-menu > .main-nav__item
   # js-main-nav > .vertical.menu.main-nav.accordion-menu > LI:nth-of-type(2)
   # gm5hu3-acc-menu > .sub-nav__item.is-submenu-item.is-accordion-submenu-item
   # gm5hu3-acc-menu > LI:nth-of-type(2)
   # js-main-nav > .vertical.menu.main-nav.accordion-menu > LI:nth-of-type(4)
   # js-main-nav > .vertical.menu.main-nav.accordion-menu > LI:nth-of-type(5)

According to the audit rules (https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules), there should be an aria role for the parent container of the existing aria roles (such as "menuitem.") 

I propose updating the aria roles so that one of the parent containers has the role "menu." I plan to tackle that this week and would welcome any feedback or assistance.

**Comment by kball** [comment]
I believe this is probably correct - we should make the accordion menu itself into a menu, since that is the role it is playing. It looks like currently the javascript is calling it a tabset, probably because it is hiding and displaying content.  However, looking at dropdown menus (which do the same), there we use menus throughout.  You should be able to fix this pretty directly in https://github.com/zurb/foundation-sites/blob/develop/js/foundation.accordionMenu.js

I'd look at the dropdown menu approach for contrast:

https://github.com/zurb/foundation-sites/blob/develop/js/foundation.dropdownMenu.js

**Comment by MichellanneLi** [comment]
Hi Kevin,

Thank you so much for your help! I took at look at those two files and
updated the aria roles specified in accordionMenu.js, changing "tablist"
and "tabpanel" to "menu" and "tabitem" to "menuitem." After making this
change, I confirmed that the accordion menu sample in my local foundation
sites documentation file (http://localhost:3000/dropdown-menu.html) was
updated with the correct aria roles by running it through the same auditing
tool that I used to discover the bug.

I sent you an separate email about a problem I'm having pushing my branch
to the remote repo.

Best Regards,

Michellanne Li
(512) 718-2207
www.michellanne.com

**Comment by kball** [comment]
This was resolved by https://github.com/zurb/foundation-sites/pull/9255 and then reworked but still fixed by https://github.com/zurb/foundation-sites/pull/9283.  Closing

**Comment by MichellanneLi** [comment]
Hi Kevin,

Thanks so much for the update! I appreciate your sharing #9238 with me so I
can learn from other developers.

Best Regards,

Michellanne Li
(512) 718-2207
www.michellanne.com

**Comment by ncoden** [comment]
@MichellanneLi I don't know if your signature with email and telephone number was intentional. I removed them, just in case. Please note that your answers by email are public.

**Comment by MichellanneLi** [comment]
@ncoden I wasn't thinking about that when I responded, so thank you so much for removing my personal information.
